src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h

```diff
@@ -0,0 +1,169 @@
+    ]],                                                     \
+    int in_alphaIsLuminance = 1                             \
+    [[                                                      \
+        string maya_attribute_name = "alphaIsLuminance",    \
+        string maya_attribute_type = "int",                 \
+        string label = "Alpha Is Luminance",                \
+        string widget = "checkBox"                          \
+    ]],                                                     
+
+#define MAYA_EFFECTS_PARAMETERS                             \
+    float in_filter = 1.0                                   \
+    [[                                                      \
+        string maya_attribute_name = "filter",              \
+        string maya_attribute_type = "float",               \
+        string label = "Filter"                             \
+    ]],                                                     \
+    float in_filterOffset = 0.0                             \
+    [[                                                      \
+        string maya_attribute_name = "filterOffset",        \
+        string maya_attribute_type = "float",               \
+        string label = "Filter Offset"                      \
+    ]],                                                     \
+    int in_invert = 0                                       \
+    [[                                                      \
+        string maya_attribute_name = "invert",              \
+        string maya_attribute_type = "bool",                \
+        string label = "Invert",                            \
+        string widget = "checkBox"                          \
+    ]]
+
+#define MAYA_UV_PARAMETERS                                  \
+    float in_uvCoord[2] = {0.0, 0.0}                        \
+    [[                                                      \
+        string maya_attribute_name = "uvCoord",             \
+        string maya_attribute_type = "float[]",             \
+        string label = "UV Coordinates"                     \
+    ]],                                                     \
+    float in_uvFilterSize[2] = {0.0, 0.0}                   \
+    [[                                                      \
+        string maya_attribute_name = "uvFilterSize",        \
+        string maya_attribute_type = "float[]",             \
+        string label = "UV Filter Size"                     \
+    ]]
+
+// Maya luminance uses Rec.601 chromaticity coordinates and Y coefficients,
+// see: https://www.itu.int/rec/R-REC-BT.601-7-201103-I/en
+float maya_luminance(color in_C)
+{
+    return as_luminance(in_C, "Rec.601");
+}
+
+void maya_colorBalance(
+    color in_colorGain,
+    color in_colorOffset,
+    color in_defaultColor,
+    float in_alphaGain,
+    float in_alphaOffset,
+    int in_invert,
+    int in_alphaIsLuminance,
+    output color out_outColor,
+    output float out_outAlpha
+    )
```

Comment:
@dictoon ok, i'll change this and update functions in other headers to follow the same convention.